dashboard/lib/build_dashboard_page.dart

```diff
@@ -109,43 +109,47 @@ class BuildDashboardPageState extends State<BuildDashboardPage> {
         child: Material(
           color: Colors.transparent,
           child: FocusTraversalGroup(
-            child: ListView(
-              children: <Widget>[
-                if (_smallScreen) ..._slugSelection(context, _buildState),
-                TextButton(
-                  child: const Text('Vacuum GitHub Commits'),
-                  onPressed: _buildState.refreshGitHubCommits,
-                ),
-                Row(
-                  children: [
-                    Expanded(child: Center(child: FilterPropertySheet(_filter))),
-                  ],
-                ),
-                Row(
-                  mainAxisSize: MainAxisSize.min,
-                  mainAxisAlignment: MainAxisAlignment.center,
-                  children: <Widget>[
-                    TextButton(
-                      child: const Text('Defaults'),
-                      onPressed: _filter!.isDefault ? null : () => _filter!.reset(),
-                    ),
-                    TextButton(
-                      child: const Text('Apply'),
-                      onPressed: _filter == _settingsBasis ? null : () => _updateNavigation(context),
-                    ),
-                    TextButton(
-                      child: const Text('Cancel'),
-                      onPressed: () {
-                        if (_filter != _settingsBasis) {
-                          _filter!.reset();
-                          _filter!.applyMap(_settingsBasis!.toMap(includeDefaults: false));
-                        }
-                        _removeSettingsDialog();
-                      },
-                    ),
-                  ],
-                ),
-              ],
+            child: FractionallySizedBox(
+              widthFactor: 0.5,
+              heightFactor: 0.7,
```

Comment:
Fractional view makes it unnecessarily large. The goldens have the dialog disappear off the bottom of the image and it's mostly blank space.

Was the original change to move from a Column to a ListView intended to keep it from overflowing a screen? The problem with ListView is that it always expands to fill its space. You can limit the main-axis (vertical) size using "shrinkwrap: true", but that doesn't stop it from filling all of the cross-axis (horizontal) space.

This widget looks like it will size itself to its children, but scroll when necessary, perhaps that is a better fit: https://api.flutter.dev/flutter/widgets/SingleChildScrollView-class.html

Reply:
We could simplify it to a sized box. There isn't much variation for the layout here, and a 300 px by 300px box is likely enough to fit the content

Reply:
Thanks a lot for the suggestions! (Wow everyone gets up so early :)

Reply:
LGTM